BREAKING.md

```diff
@@ -20,8 +20,13 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 ## 2.0.0-internal.3.0.0 Upcoming changes
 
 ## 2.0.0-internal.3.0.0 Breaking changes
+- [Make existing flag required](#make-existing-flag-required)
```

Comment:
As a general note, this title should be specific and call out the interface and method where this is changed.

This title is very generic. It should say something like "existing parameter is now required in IRuntimeFactory::intantiateRuntime".